build.psm1

```diff
@@ -137,6 +137,25 @@ function Copy-CompatibilityProfiles
     Copy-Item -Force $profileDir/* $targetProfileDir
 }
 
+function Start-ReferenceBuild
+{
+    try {
+        Push-Location Reference
+        $buildOutput  = dotnet build --configuration Release
+        if ( ! $? ) {
```

Comment:
A style preference but I find the `-not` operator quite readable, especially when there are already a lot of symbols involved in this expression

Reply:
fixed